### PR TITLE
Update ShortLinkSql methods to include meta tags

### PR DESCRIPTION
**Pull request by kapillamba4**
* Add MetaTags entity.
* Update ShortLink entity and embed MetaTags entity in it.
* Update ShortLinkSql methods to include meta tags.
* Update Integration Tests.

Part of #809

### CODE REVIEW

File: backend/app/adapter/sqldb/table/short_link.go [hunk 1]
```diff
@@ -2,17 +2,29 @@ package table
 
 // ShortLink represents database table columns for 'short_link' table
 var ShortLink = struct {
-	TableName       string
-	ColumnAlias     string
-	ColumnLongLink  string
-	ColumnCreatedAt string
-	ColumnExpireAt  string
-	ColumnUpdatedAt string
+	TableName                string
```
[the remark this answers is not in the document]
Comment (kapillamba4) [reply backend/app/adapter/sqldb/table/short_link.go:5]: @arberiii 
**CreateShortLink**, **GetShortLinkByAlias**, **GetShortLinksByAliases** all require meta tags. There will be an option to edit meta tags for each alias from table. `GetShortLinkByAlias` will be used in ssr to get meta tags for facebook/twitter/linkedin sharing. In `CreateShortLink` meta tags will be provided by user which can later be customized.

[the remark this answers is not in the document]
Comment (arberiii) [reply backend/app/adapter/sqldb/table/short_link.go:5]: I see, thank you for clarifying.

File: backend/app/adapter/sqldb/short_link_integration_test.go [hunk 2]
```diff
@@ -16,14 +16,20 @@ import (
...
 	table.ShortLink.ColumnUpdatedAt,
+	table.ShortLink.ColumnOGTitle,
+	table.ShortLink.ColumnOGDescription,
+	table.ShortLink.ColumnOGImageURL,
+	table.ShortLink.ColumnTwitterTitle,
+	table.ShortLink.ColumnTwitterDescription,
+	table.ShortLink.ColumnTwitterImageURL,
 )
 
 type shortLinkTableRow struct {
```
Comment (Coteh) [inline backend/app/adapter/sqldb/short_link_integration_test.go:35]: Test struct should be updated to include the new fields. I get the following errors when I run integration test script:
```
app/adapter/sqldb/short_link_integration_test.go:117:6: unknown field 'ogTitle' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:118:6: unknown field 'ogDescription' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:119:6: unknown field 'ogImageURL' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:120:6: unknown field 'twitterTitle' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:121:6: unknown field 'twitterDescription' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:122:6: unknown field 'twitterImageURL' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:130:6: unknown field 'ogTitle' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:131:6: unknown field 'ogDescription' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:132:6: unknown field 'ogImageURL' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:133:6: unknown field 'twitterTitle' in struct literal of type shortLinkTableRow
```

Comment (Coteh) [reply backend/app/adapter/sqldb/short_link_integration_test.go:35]: My bad, it's in #830.

File: backend/app/adapter/sqldb/short_link.go [hunk 3]
```diff
@@ -94,14 +106,20 @@ WHERE "%s"=$5;`,
...
 FROM "%s" 
 WHERE "%s"=$1;`,
 		table.ShortLink.ColumnAlias,
 		table.ShortLink.ColumnLongLink,
 		table.ShortLink.ColumnExpireAt,
 		table.ShortLink.ColumnCreatedAt,
 		table.ShortLink.ColumnUpdatedAt,
+		table.ShortLink.ColumnOGTitle,
+		table.ShortLink.ColumnOGDescription,
+		table.ShortLink.ColumnOGImageURL,
```
Comment (byliuyang) [inline backend/app/adapter/sqldb/short_link.go:119]:
```suggestion
		table.ShortLink.ColumnOpenGraphTitle,
		table.ShortLink.ColumnOpenGraphDescription,
		table.ShortLink.ColumnOpenGraphImageURL,
```

**Comment by byliuyang** [comment]
Hi @kapillamba4 , thank you so much for working on this PR! Please address comments from codeclimate!

**Pull request merged**